components/html-block/html-block.js

```diff
@@ -135,6 +135,14 @@ class HtmlBlock extends LitElement {
 		`];
 	}
 
+	connectedCallback() {
+		super.connectedCallback();
+		if (!this._templateObserver) return;
```

Comment:
If the observer doesn't already exist, we don't need to do anything here, since presumably `firstUpdated` hasn't yet run.